src/java.base/share/classes/java/lang/reflect/code/op/ExtendedOp.java

```diff
@@ -3078,6 +3092,38 @@ public TypeElement resultType() {
             }
         }
 
+        @OpFactory.OpDeclaration(MatchAllPatternOp.NAME)
+        public static final class MatchAllPatternOp extends PatternOp {
+
+            public static final String NAME = "pattern.match.all";
+
+            MatchAllPatternOp(ExternalizedOp def) {
+                super(def);
+            }
+
+            MatchAllPatternOp(MatchAllPatternOp that, CopyContext cc) {
+                super(that, cc);
+            }
+
+            MatchAllPatternOp() {
+                super(NAME, List.of());
+            }
+
+            public static MatchAllPatternOp create(ExternalizedOp def) {
+                return new MatchAllPatternOp(def);
+            }
```

Comment:
You don't need this method, the op factory can use `ExternalizedOp` constructor, just make it pubic.

I think we may be able to merge the `create` methods and the constructor when we can use flexible constructor bodies ([JEP 482](https://openjdk.org/jeps/482)).